src/vm/tests/datamaps.rs

```diff
@@ -56,6 +56,83 @@ fn test_simple_tea_shop() {
     assert_executes(expected, test1);
 }
 
+#[test]
+fn test_explicit_syntax_tuple() {
```

Comment:
It would be good to add another kind of "explicit syntax test" using a `let` bound variable, e.g.:

```
(let ((my-tuple (tuple (key 1))))
   (fetch-entry! kv-store my-tuple))
```

Reply:
Addressed with `fn test_bound_tuple()`